include/dpp/channel.h

```diff
@@ -123,6 +147,15 @@ class channel : public managed {
 	/** Permission overwrites to apply to base permissions */
 	std::vector<permission_overwrite> permission_overwrites;
 
+	/** Approximate count of messages in a thread (threads) */
+	uint8_t message_count;
+
+	/** Approximate count of members in a thread (threads) */
+	uint8_t member_count;
```

Comment:
Is this big enough? does discord allow > 255 members in a thread? perhaps uint16_t?

Reply:
yes, it's big enough 
![image](https://user-images.githubusercontent.com/72332327/123706318-fcfc2900-d81c-11eb-8345-16f10b6ccc66.png)